packages/common/src/utils/mathUtils.ts

```diff
@@ -77,3 +77,7 @@ export const isZero = (v: Vector3): boolean => {
 export const normalizeRange = (val, rangeMin, rangeMax) => {
   return (val - rangeMin) / (rangeMax - rangeMin)
 }
+
+export const approxeq = (v1, v2, epsilon = 0.001) => {
```

Comment:
can you rename to `equalsEpsillon` for clarity? Also, v1 and v2 parameters should be typed.